scripts/experiment_1/nbconverted/1_train_vae.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Train VAE
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import ast
+import pandas as pd
+import numpy as np
+import random
+import warnings
+warnings.filterwarnings(action='ignore')
+
+from functions import vae
+
+from numpy.random import seed
+randomState = 123
+seed(randomState)
+
+
+# In[2]:
+
+
+# Create list of base directories
+
+base_dir = os.path.abspath(os.path.join(os.getcwd(),"../.."))
+
+base_dirs = [os.path.join(base_dir, "data"),
+             os.path.join(base_dir, "models"),
+             os.path.join(base_dir, "output"),
+             os.path.join(base_dir, "data", "encoded"),
+             os.path.join(base_dir, "output", "stats"),
+             os.path.join(base_dir, "output", "viz")
+             ]
+
+# Check if analysis directory exist otherwise create
+
+for each_dir in base_dirs:
+
+    if os.path.exists(each_dir):
+        print('directory already exists: {}'.format(each_dir))
```

Comment:
With newer python3, you can just call 

`print(f' dir exists {each_dir}.')`

Reply:
I am using py3.5, I don't think this syntax is supported until py3.6  Getting an error so just leaving it in its original format for now.